pkg/scheduler/queue/multi_queuing_algorithm_tree_queue_benchmark_test.go

```diff
@@ -375,39 +375,45 @@ func TestMultiDimensionalQueueAlgorithmSlowConsumerEffects(t *testing.T) {
 	for _, weightedQueueDimensionTestCase := range weightedQueueDimensionTestCases {
 		numTenants := len(weightedQueueDimensionTestCase.tenantQueueDimensionsWeights)
 
-		tqa := newTenantQuerierAssignments()
+		tqaNonFlipped := newTenantQuerierAssignments()
+		tqaFlipped := newTenantQuerierAssignments()
+		tqaQuerierWorkerPrioritization := newTenantQuerierAssignments()
```

Comment:
could just initialize them right in the tree constructor the same way we do the round robin states since we never reference them again, but nbd

Reply:
We do reference them again in the `treeScenarios` initialization